### PR TITLE
RHMAP-10124 update to Swift 3

### PR DESCRIPTION
**Pull request by jcesarmobile**

### CODE REVIEW

**Comment by aidenkeating** [comment]
@jcesarmobile LGTM 👍  Like you said, waiting for the release of 5, but I did verify it too. Image below.

[FH Init Success Message](https://cloud.githubusercontent.com/assets/8698527/20393597/5fe7a590-acd4-11e6-8bc1-a2cc77c3d068.png)

**Comment by jcesarmobile** [comment]
I've fixed the FeedHenry SDK and squashed the commits.
@aidenkeating @corinnekrych can you take a final look?

File: blank-ios-app.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -348,15 +348,15 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "org.feedhenry.blank-ios-app";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 2.3;
+				SWIFT_VERSION = 3.0;
```
Comment (corinnekrych) [inline blank-ios-app.xcodeproj/project.pbxproj]: as discussed 3.0.1 works better for Xcode8.1

**Comment by corinnekrych** [comment]
@jcesarmobile  Apart from small 3.0.1 comment, good to merge

**Comment by jcesarmobile** [comment]
@corinnekrych done

**Pull request merged**